cmd/remove/remove.go

```diff
@@ -74,7 +78,7 @@ webman remove rg`,
 			color.HiBlack("No packages selected for removal.")
 			return nil
 		}
-		pkgConf, err := pkgparse.ParsePkgConfigLocal(pkg, false)
+		pkgConf, err := pkgparse.ParsePkgConfigLocal(cfg.PkgRepos, pkg)
```

Comment:
How does this handle duplicate pkg config titles across PkgRepos? If I have a `go` pkg in the main repo, but you have a `go` pkg in your repo, which will be deleted? Is there priority set?

Reply:
The priority is whichever it finds first. Because the config is a list, you can semi-manually prioritize based on which one is first in that list.